DipSandhi-eatingplace.md

```diff
@@ -0,0 +1,5 @@
+
+### Type of food to eat
+  - Itallian
+  - Indian
```

Comment:
@Diptikanta Please be more specific.  It is unclear which type of Indian you are referring to.  We could have avoided this confusion if Christopher Columbus would have known where he actually landed, but the reality is he has left us with this technical debt of identification that we've had to deal with for the past 500 years...